src/Model/SimpleRepositoryLoader.php

```diff
@@ -14,12 +14,15 @@
 
 class SimpleRepositoryLoader implements IRepositoryLoader
 {
-	/** @var IRepository[] */
+	/**
+	 * @var IRepository[]
+	 * @phpstan-var array<class-string<IRepository<\Nextras\Orm\Entity\IEntity>>, IRepository<\Nextras\Orm\Entity\IEntity>>
+	 */
 	private $repositories;
 
 
 	/**
-	 * @param IRepository[] $repositories
+	 * @phpstan-param list<IRepository<\Nextras\Orm\Entity\IEntity>> $repositories
```

Comment:
Does phpstan know what is `list`? I think it needs to be `array<int, IRepository<\Nextras\Orm\Entity\IEntity>>`

Reply:
hm, I would better try it myself than write publicly 😄 
ok it works, thx for new info!

Reply:
https://phpstan.org/r/6051a535-a402-42fa-af23-57ca3565630f